yaksh/models.py

```diff
@@ -54,6 +54,8 @@
         ("integer", "Answer in Integer"),
         ("string", "Answer in String"),
         ("float", "Answer in Float"),
+        ("arrange", "Arrange Options in Order"),
```

Comment:
No. I don't believe the alternative provided is any better than the original name. Technically an option can be defined as a thing that may or may not be chosen. Hence when I say `Arrange options in order`, I emphasize that you may or may not choose an **option** to arrange in a particular order. But I do agree that the word **correct** should be included so that the name becomes `Arrange Options in Correct Order.`

Reply:
Alright, but once confirm with Sir.

Reply:
I think just "Arrange in Correct Order" is fine as these are not technically options!